### ISSUE TITLE
Error with image names with special characters

### ISSUE THREAD

**Issue by tomjohnneill** [scanny/python-pptx#223]
Returns error:
`File "src/lxml/parser.pxi", line 1819, in lxml.etree._parseMemoryDocument (src\lxml\lxml.etree.c:116494)
  File "src/lxml/parser.pxi", line 1700, in lxml.etree._parseDoc (src\lxml\lxml.etree.c:115040)
  File "src/lxml/parser.pxi", line 1040, in lxml.etree._BaseParser._parseUnicodeDoc (src\lxml\lxml.etree.c:109165)
  File "src/lxml/parser.pxi", line 573, in lxml.etree._ParserContext._handleParseResultDoc (src\lxml\lxml.etree.c:103404)
  File "src/lxml/parser.pxi", line 683, in lxml.etree._handleParseResult (src\lxml\lxml.etree.c:105058)
  File "src/lxml/parser.pxi", line 613, in lxml.etree._raiseParseError (src\lxml\lxml.etree.c:103967)
  lxml.etree.XMLSyntaxError: EntityRef: expecting ';', line 3, column 31`

**Comment by scanny** [comment]
You'll need to provide more details, in particular an example of an offending file name.

**Comment by sreejeet** [comment]
I ran into a very similar issue when using slide.shapes.add_picture() with an image file that had an ampersand (&) in it.

Adding image "Amrat - Medical & General Stores 2.jpeg"
  File "/home/redacted/anaconda3/envs/ag/lib/python3.7/site-packages/pptx/oxml/__init__.py", line 40, in parse_xml
    root_element = etree.fromstring(xml, oxml_parser)
  File "src/lxml/etree.pyx", line 3234, in lxml.etree.fromstring
  File "src/lxml/parser.pxi", line 1876, in lxml.etree._parseMemoryDocument
  File "src/lxml/parser.pxi", line 1757, in lxml.etree._parseDoc
  File "src/lxml/parser.pxi", line 1068, in lxml.etree._BaseParser._parseUnicodeDoc
  File "src/lxml/parser.pxi", line 601, in lxml.etree._ParserContext._handleParseResultDoc
  File "src/lxml/parser.pxi", line 711, in lxml.etree._handleParseResult
  File "src/lxml/parser.pxi", line 640, in lxml.etree._raiseParseError
  File "<string>", line 3
lxml.etree.XMLSyntaxError: xmlParseEntityRef: no name, line 3, column 62

**Comment by mccannjb** [comment]
I have also confirmed this behavior is exhibited in python-pptx version 0.6.18 when calling the slide.shapes.add_picture() method with a filename containing an ampersand (&).

During slide generation, passing a filepath with an ampersand to the add_picture() method results in a lxml.etree.XMLSyntaxError:
```
from pptx import Presentation

image_to_add = 'image_with_&_in_name.png'
prs = Presentation.Pres()
slide = prs.slides.add_slide(prs.slide_layouts[0])
shapes = slide.shapes

img = shapes.add_picture(image_to_add)    # This raises a lxml.etree.XMLSyntaxError (xmlParseEntityRef: no name, ...)
```
A simple, one-line workaround for this issue is to load the image to memory and pass that object to add_picture():
```
from pptx import Presentation

image_to_add = 'image_with_&_in_name.png'
prs = Presentation.Pres()
slide = prs.slides.add_slide(prs.slide_layouts[0])
shapes = slide.shapes

with open(image_to_add, 'rb') as loaded_image:
    img = shapes.add_picture(loaded_image)    # This completes successfully
```

**Comment by antoonhuiskens** [comment]
similar here: filename "Q&A-bubbles-300x300-PMS355.png" gives

 lxml.etree.XMLSyntaxError: EntityRef: expecting ';', line 3, column 75

**Comment by beyarkay** [comment]
Same here, I also got issues with a name like:
```img&.png```

The error message is (with private info redacted):
```
Traceback (most recent call last):
  File "myscript.py", line 1057, in <module>
    main()
  File "myscript.py", line 122, in main
    write_to_pptx(f"{SAVE_DIR}/Graphs-Complete")
  File "myscript.py", line 685, in write_to_pptx
    pic = slide.shapes.add_picture("img&.png", Cm(0), Cm(3), width=ppt.slide_width)
  File ".../python3.7/site-packages/pptx/shapes/shapetree.py", line 295, in add_picture
    pic = self._add_pic_from_image_part(image_part, rId, left, top, width, height)
  File ".../python3.7/site-packages/pptx/shapes/shapetree.py", line 399, in _add_pic_from_image_part
    pic = self._grpSp.add_pic(id_, name, desc, rId, x, y, scaled_cx, scaled_cy)
  File ".../python3.7/site-packages/pptx/oxml/shapes/groupshape.py", line 81, in add_pic
    pic = CT_Picture.new_pic(id_, name, desc, rId, x, y, cx, cy)
  File ".../python3.7/site-packages/pptx/oxml/shapes/picture.py", line 70, in new_pic
    pic = parse_xml(xml)
  File ".../python3.7/site-packages/pptx/oxml/__init__.py", line 40, in parse_xml
    root_element = etree.fromstring(xml, oxml_parser)
  File "src/lxml/etree.pyx", line 3237, in lxml.etree.fromstring
  File "src/lxml/parser.pxi", line 1896, in lxml.etree._parseMemoryDocument
  File "src/lxml/parser.pxi", line 1777, in lxml.etree._parseDoc
  File "src/lxml/parser.pxi", line 1082, in lxml.etree._BaseParser._parseUnicodeDoc
  File "src/lxml/parser.pxi", line 615, in lxml.etree._ParserContext._handleParseResultDoc
  File "src/lxml/parser.pxi", line 725, in lxml.etree._handleParseResult
  File "src/lxml/parser.pxi", line 654, in lxml.etree._raiseParseError
  File "<string>", line 3
lxml.etree.XMLSyntaxError: xmlParseEntityRef: no name, line 3, column 49

```

Is this project dead? or alternatively is it still accepting pull requests?

I'd be happy to add a more comprehensible error message, and possibly to fix the root cause.

**Comment by scanny** [comment]
Fixed in v0.6.20 due out here tonight or tomorrow.
